src/Cookie/CookieJar.php

```diff
@@ -256,7 +259,7 @@ private function getCookiePathFromRequest(RequestInterface $request): string
         if ('/' === $uriPath) {
             return '/';
         }
-        if (0 === $lastSlashPos = \strrpos($uriPath, '/')) {
+        if (0 === ($lastSlashPos = \strrpos($uriPath, '/')) || false === $lastSlashPos) {
```

Comment:
```suggestion
        $lastSlashPos = \strrpos($uriPath, '/');
        if (0 === $lastSlashPos || false === $lastSlashPos) {
```

Reply:
but isn't this changing the behavior here? 

https://3v4l.org/rZZ4s